assets/src/blocks/TopicLink/TopicLinkBlock.js

```diff
@@ -0,0 +1,59 @@
+import {TopicLinkEditor} from './TopicLinkEditor.js';
+
+const {registerBlockType, getBlockTypes} = wp.blocks;
+const {__} = wp.i18n;
+
+const BLOCK_NAME = 'planet4-blocks/topic-link';
+
+export const registerTopicLinkBlock = () => {
+  const blockAlreadyExists = getBlockTypes().find(block => block.name === BLOCK_NAME);
+
+  if (blockAlreadyExists) {
+    return;
+  }
+
+  registerBlockType(BLOCK_NAME, {
+    title: 'Topic Link',
+    description: __('', 'planet4-blocks-backend'),
```

Comment:
Is the description mandatory? If it's not I would just remove it, if it is it doesn't need translation for an empty string

Reply:
Fixed!